test/read-file-test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const expect = require('chai').expect;
+const readFile = require('../lib/read-file.js');
+
+describe('Read File', function() {
+  describe('returns correct contents in order', function() {
+    it('should return the contents of the file in the defined order', function(done) {
+      var readFileArray = ['412063616e6e6572', '412063616e6e6572', '4275742061206361'];
+      readFile(function(data) {
```

Comment:
Way to mock the bytes of each file in an array and using it to test that your files are being read and logged in order.